packages/react-native-bpk-component-badge/src/BpkBadgeIcons.js

```diff
@@ -63,6 +63,7 @@ const BpkBadgeIcons = (props: Props) => {
         <Fragment key={`${icon.props.icon}`}>
           {icon}
           {separator && index < adjustedIcons.length - 1 && (
+            // $FlowFixMe
```

Comment:
Can we include a comment with more details when we use `$FlowFixMe`?

Reply:
Yeah of course, I just have to remember why ... 😂

Reply:
I removed the `// $FlowFixMe` so I could see what the error was, and I'm none the wiser..

![screen shot 2019-02-21 at 11 29 08](https://user-images.githubusercontent.com/73652/53165965-f8c7fd00-35cb-11e9-94c4-ecdb64f6327c.png)

Reply:
If I remove `StyleSheet.flatten` from the styles being applied to that `BpkText` component then it goes away 🔮

Reply:
![](https://media1.tenor.com/images/11dfa88cf0194f13497cb7fb13af3b12/tenor.gif?itemid=9490827)